app/services/idv/steps/verify_base_step.rb

```diff
@@ -56,6 +55,15 @@ def add_proofing_costs(results)
         end
       end
 
+      def pii
+        flow_session[:pii_from_doc].presence || flow_session[:pii_from_user]
```

Comment:
Are we confident that there's no way that `flow_session[:pii_from_doc]` will be defined when we're in the in person flow?

Reply:
It may be defined (in fact, I think it will be) but should remain empty

Reply:
Interestingly I _think_ [this](https://github.com/18F/identity-idp/blob/980028a222cdcf5a17ca4c8eb6fe87aaffbb0f99/app/services/doc_auth/response.rb#L11) is where it originally gets defined. It was hard to track down where it gets set